src/test/java/seedu/address/model/AddressBookTest.java

```diff
@@ -83,6 +84,21 @@ public void getPersonList_modifyList_throwsUnsupportedOperationException() {
         assertThrows(UnsupportedOperationException.class, () -> addressBook.getPersonList().remove(0));
     }
 
+    @Test
+    public void syncMeetingTimes() {
+        Person alpha = new PersonBuilder().withMeetingTimes("28-07-2020-15:00").build();
+        Person beta = new PersonBuilder().withMeetingTimes().build();
+        List<Person> persons = Arrays.asList(CARL, alpha);
+        List<Person> newPersons = Arrays.asList(CARL, beta);
+        AddressBook newAddressBook = new AddressBook();
+        newAddressBook.setPersons(persons);
+        AddressBook expectedAddressBook = new AddressBook();
+        expectedAddressBook.setPersons(newPersons);
+        newAddressBook.syncMeetingTimes();
+
+        assertEquals(newAddressBook, expectedAddressBook);
+    }
+
```

Comment:
Not sure if we have to test boundary values, that would be troublesome as would need to somehow create a stub LocalDateTime to represent the 'current' time.